### DISCUSSION TITLE
Keycloak vulnerability CVE-2022-25844

### DISCUSSION THREAD

**Discussion by renovk** [keycloak/keycloak#13119] [Q&A]
Any version in keycloak has vulnerability  fixed CVE-2022-25844 or how to fix that in current version 18 legacy

**Comment by abstractj** [comment]
Hi @renovk thanks for bringing this issue to our attention, we actually upgraded angular to 1.8.3. If I understood it correctly, currently we don't have any exploits for the issue mentioned and there's no fix provided by the Angular team. The default recommendation is to replace the component, probably by `angular/core` which would break backwards compatibility.

That being said, I'd suggest as a mitigation to use our new admin console with React. The old admin console with Angular.js will be deprecated soon. We will continue to maintain the old console, as soon as the Angular team publishes a new release with the fix.

**Reply by sam-lin-privoro** [reply]
Thanks @abstractj ! Which keycloak version will have this angular 1.8.3 upgrade?

**Comment by christian-2** [comment]
@abstractj FYI there is still a reference to Angular 1.4.4 in [`docs/dependency-license-information.md`](https://github.com/keycloak/keycloak/blob/main/docs/dependency-license-information.md). This may be distracting and it may be worthwhile to version-bump it.

**Reply by jonkoops** [reply]
We actually no longer have Angular JS as a dependency, so this can be removed entirely.
